[CIS GCP] Notify the user about cloud shell editor deprecation (#5031)
elastic/security-docs: docs/cloud-native-security/cspm-get-started-gcp.asciidoc

+
image::images/cspm-cloudshell-trust.png[The cloud shell confirmation popup]
+

> NOTE: Google has deprecated its old Cloud Shell editor. If you continue to use it, you may encounter the following pop-up:
> +
> image::images/cspm-cloudshell-old-editor.png[The cloud shell switch editor popup]
> +
> If the popup appears, click **X** or **Try the new editor** and follow the next steps. When you switch to the new editor, your context should remain unchanged.

Reviewer: Inconsistency between `pop-up` and `popup`, but we could just call it a "message" instead. The fact that it pops up isn't super important.
```suggestion
NOTE: Google has deprecated its old Cloud Shell editor. If you continue to use it, you may encounter the following message:
+
image::images/cspm-cloudshell-old-editor.png[The cloud shell switch editor popup]
+
If the message appears, click **X** or **Try the new editor** and follow the next steps. When you switch to the new editor, your context should remain unchanged.
```

> If the popup appears, click **X** or **Try the new editor** and follow the next steps. When you switch to the new editor, your context should remain unchanged.

Reviewer 2: ```suggestion
If the popup appears, click **X** or **Try the new editor** and follow the next steps. When you switch to the new Editor, your context should remain unchanged.
```

> If the message appears, click **X** or **Try the new editor** and follow the next steps. When you switch to the new editor, your context should remain unchanged.

Reviewer 3: ```suggestion
If the message appears, click **X** or **Try the new Editor** and follow the next steps. When you switch to the new editor, your context should remain unchanged.
```

. In Google Cloud Shell, execute the command you copied. Once it finishes, return to {kib} and wait for the confirmation of data received from your new integration. Then you can click **View Assets** to see your data.

> NOTE: If you encounter any issues running the command, return to Kibana and navigate again to Google Cloud Shell.

Reviewer: ```suggestion
NOTE: If you encounter any issues running the command, return to {kib} and navigate again to Google Cloud Shell.
```
